omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -91,7 +107,15 @@ public Object create(SimpleObject propertiesToCreate, RequestContext context) th
 		
 		T delegate = handler.newDelegate();
 		setConvertedProperties(delegate, propertiesToCreate, handler.getCreatableProperties(), true);
-		ValidateUtil.validate(delegate);
+		
+		try {
+			ValidateUtil.validate(delegate);

```

Comment:
I think save(delegate) already invokes the validator, so no need to invoke it twice.